proto/faas/v1/faas.proto

```diff
@@ -0,0 +1,130 @@
+syntax = "proto3";
+package nitric.faas.v1;
+
+// protoc plugin options for code generation
+option go_package = "nitric/v1;v1";
+option java_package = "io.nitric.proto.faas.v1";
+option java_multiple_files = true;
+option java_outer_classname = "NitricFaas";
+option php_namespace = "Nitric\\Proto\\Faas\\V1";
+option csharp_namespace = "Nitric.Proto.Faas.v1";
+
+// Service for management of event topics
```

Comment:
need to update the comments to match this service and its messages, a few copy/paste hangovers are left behind.

Reply:
fixed.